src/App.tsx

```diff
@@ -1,18 +1,48 @@
-import React from 'react';
+import React, { ChangeEvent, useEffect, useState } from 'react';
 import './App.css';
+import {
+  Routes, Route, useLocation,
+} from 'react-router-dom';
 import { getNumbers } from './utils';
+import { Content } from './components/Content/Content';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const { search } = useLocation();
+  const page = new URLSearchParams(search).get('page') || 1;
+  const perPage = new URLSearchParams(search).get('perPage') || 5;
```

Comment:
It is better to set `5` to variable, to make it easier to change this limit in future